parsing.indexing/test/unit/src/org/netbeans/modules/parsing/impl/indexing/InjectedTasksSupportTest.java

```diff
@@ -359,21 +379,26 @@ public int getIndexVersion() {
 
     private static final class CIHandler extends Handler {
 
+        private static final Logger LOG = Logger.getLogger(CIHandler.class.getName());
 
         enum Event {
             ENQUEUE,
             EXEXUTE
         };
-        
+
         private final Set<Event> expectedEvents = EnumSet.noneOf(Event.class);
 
-        public synchronized void expect (@NonNull final Event... events) {
+        public synchronized void expect(@NonNull final Event... events) {
+            LOG.info(String.format("Current expected events: %s", this.expectedEvents));
+            LOG.info(String.format("New events: %s", Arrays.toString(events)));
             expectedEvents.addAll(Arrays.asList(events));
+            LOG.info(String.format("Updated expected events: %s", this.expectedEvents));
         }
 
         @Override
         public synchronized void publish(LogRecord record) {
             final String message = record.getMessage();
+            LOG.info(String.format("Record message: %s", message));
```

Comment:
This should expand the message with the parameters. I needed to look twice to see why I saw {0} in the output.